### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by chineduG**
Corrections of misspelled words.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -350,11 +350,11 @@ class Animal extends Organism {
 **Major changes**:
 
 * Builders now use `built_value`, and have a more consistent, friendly API.
-* Builders are now consistent - they don't any work until code is emitted.
+* Builders are now consistent - they don't work until code is emitted.
```
Comment (devoncarew) [inline CHANGELOG.md:353]: This should probably read `they don't perform work until...` or `they don't do any work until...`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -621,7 +621,7 @@ literal(true).asAssign(reference('flag'))
 
 ## 1.0.0-alpha+7
 
-- Make use of new analyzer API in preparation for analyzer version 0.30.
+- Make use of the new analyzer API in preparation for analyzer version 0.30.
```
Comment (devoncarew) [inline CHANGELOG.md:624]: `use of new analyzer APIs`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -746,3 +746,5 @@ automatically added to a `LibraryBuilder` for you if you use
 ## 0.1.0
 
 - Initial version
+
+
```
Comment (devoncarew) [inline CHANGELOG.md:750]: nit: extra training eol

**Review by devoncarew — COMMENTED** [review]
One main comment around the builder verbiage -

File: CHANGELOG.md (hunk 1 above)
Comment (natebosch) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (natebosch) [reply CHANGELOG.md]: Done

**Review by devoncarew — APPROVED** [review]

**Pull request merged**